mappings/mappings.php

```diff
@@ -8,741 +8,1196 @@
+            'weibo'                               => 'weiboapp',
             'wetab'                               => 'wetabbrowser',
+            'whale'                               => 'whalebrowser',
             'whatsapp'                            => 'whatsappbot',
             'whatweb'                             => 'whatwebwebscanner',
             'whitehataviator'                     => 'aviator',
+            'willcom'                             => null,
+            'windowsrssplatorm'                   => 'windowsrssplatform',
             'windowsrssreader'                    => 'windowsrssplatform',
+            'wire'                                => 'wireapp',
+            'wmplayer'                            => 'windowsmediaplayer',
+            'woobot'                              => 'woorank',
+            'wordpresscom'                        => 'wordpress',
+            'wosbrowser'                          => 'webkitwebos',
+            'wscheckcom'                          => 'wscheckbot',
+            'xbox360'                             => 'internetexplorer',
+            'xboxone'                             => 'internetexplorer',
             'xenulinksleuth'                      => 'xenuslinksleuth',
             'xerka'                               => 'xerkawebbot',
+            'yabrowser'                           => 'yandexbrowser',
             'yahoojapan'                          => 'yjagent',
-            'yahoomobilemail'                     => 'yahoomailmobile',
             'yahoommaudvid'                       => 'mmaudvidcrawler',
+            'yahoomobilemail'                     => 'yahoomailmobile',
             'yandex'                              => 'yandexbrowser',
+            'yandexcalendar'                      => 'yandexcalendarbot',
+            'yandeximages'                        => 'yandeximagesearch',
+            'yandexmobile'                        => 'yandexbrowser',
             'yandexsearch'                        => 'yandexsearchapp',
+            'yandexsitelinks'                     => 'yandexsitelinksbot',
+            'yandexvertis'                        => 'yandexvertisbot',
+            'yeti'                                => 'naverbot',
+            'yfsbjcrawler'                        => 'yjagent',
             'yfsjcrawler'                         => 'yjagent',
             'yjbri'                               => 'yjagent',
             'yjbrw'                               => 'yjagent',
-            'ypcrawl'                             => 'yellowpages',
             'yodao'                               => 'yodaobot',
+            'ypcrawl'                             => 'yellowpages',
             'zao'                                 => 'zaocrawler',
+            'zetakey'                             => 'zetakeybrowser',
             'zeus'                                => 'zeusbot',
+            'zioxz223wapbrowser'                  => null,
```

Comment:
This could probably be a generic WAP Browser no?

Reply:
removed this mapping, result looks like an error in the parser